subprojects/common/file-storage/src/main/kotlin/com/avito/filestorage/SettableFuture.kt

```diff
@@ -5,6 +5,12 @@ import java.util.concurrent.atomic.AtomicBoolean
 
 interface FutureValue<T> {
     fun get(): T
+
+    companion object {
+        fun <T> stub(stubValue: T): FutureValue<T> = object : FutureValue<T> {
```

Comment:
`stub` sounds like a test stub. 
Maybe we could find an alternative name: just, of, from, ...